.github/workflows/build.yml

```diff
@@ -0,0 +1,165 @@
+name: google drive
+
+on:
+  push:
+    branches:
+      - master
+      - 'SP/**'
+      - 'HF/**'
+      - company_release
+      - 'GOOGLEDOCS**'
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      commitMessage:
+        description: Enter the Commit Message
+        required: true
+env:
+  SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  QUAY_PASSWORD:  ${{ secrets.QUAY_PASSWORD }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  GITHUB_EVENT: ${{ github.event_name }}
+  BRANCH_NAME: ${{ github.ref_name }}
+
+jobs:
+  Run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        stage: [ build , source_clear, e2e_tests ]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: adopt
+          java-version: 17
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+
+      - name: Setup maven
+        shell: bash
+        run: |
+          [ -d ${HOME}/.m2/ ] || mkdir -p ${HOME}/.m2/
+          cp -v _ci/settings.xml ${HOME}/.m2/ || cp -v .travis.settings.xml ${HOME}/.m2/settings.xml
+
+      - name: Cache the Maven packages to speed up build
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: Run ${{ matrix.stage }}
+        if: github.ref_name != 'company_release'
+        run: |
+           bash _ci/init.sh
+           bash _ci/${{ matrix.stage }}.sh
+
+  Release:
+    if: ${{ ( startsWith(github.ref_name, 'HF/')  || startsWith(github.ref_name, 'SP/') || github.ref_name == 'master' )  &&
+        contains(github.event.head_commit.message, '[trigger release]') || inputs.commitMessage == '[trigger release]'  &&
+        github.event_name != 'pull_request' }}
```

Comment:
In the Travis.yml we don't have this condition to check if event in not pull request.